### PR TITLE
Feat: Player death

### PR DESCRIPTION
**Pull request by raulbatista94**
## Changes
- Add new player atlas for death
- Add life indicators
- Add player death

### CODE REVIEW

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift [hunk 1]
```diff
@@ -122,5 +122,9 @@ private extension ShootingPumpkin {
         projectile.node?.removeFromParent()
         
         print("Projectile collision with \(body)")
```
Comment (robha141) [inline SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift]: you can remove this print 😀

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift [hunk 2]
```diff
@@ -122,5 +122,9 @@ private extension ShootingPumpkin {
...
+        
+        if body == levelScene?.player.physicsBody {
+            levelScene?.player.hit()
+        }
```
Comment (robha141) [inline SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift]:
```suggestion
        guard body == levelScene?.player.physicsBody else {
            return
        }
        
        levelScene?.player.hit()
```

**Review by robha141 — APPROVED** [review]
🐉

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift (hunk 1 above)
Comment (raulbatista94) [reply SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift]: Removed 👍

File: SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift (hunk 2 above)
Comment (raulbatista94) [reply SpriteKit/MendeluSpriteKit/Game/SceneObjects/ShootingPumpkin.swift]: Changed thanks 👍

**Review by tinaStremenova — APPROVED** [review]
🍓

**Pull request merged**